src/app/components/navigation/SharedToolbar.tsx

```diff
@@ -164,9 +164,7 @@ export const SharedToolbar = (props: SharedToolbarProps): JSX.Element => {
                             secondary={subtitle}
                         />
                     ) : (
-                        <Typography>
-                            Power Xpert <b>Blue</b>
-                        </Typography>
+                        <Typography>Brightlayer UI</Typography>
```

Comment:
The styling is now gone 

![image](https://user-images.githubusercontent.com/8997218/144139738-da25767e-a4ae-44d7-97ad-65c8d66d7561.png)

![image](https://user-images.githubusercontent.com/8997218/144139753-9431b1dd-d094-4bed-92e6-79aa29a61806.png)

Spell out the Brightlayer UI, and bolden the "User Interface" part